libs/log/default.go

```diff
@@ -76,21 +76,41 @@ func (l defaultLogger) Info(msg string, keyVals ...interface{}) {
 }
 
 func (l defaultLogger) Error(msg string, keyVals ...interface{}) {
-	e := l.Logger.Error()
-
-	e.Fields(getLogFields(keyVals...)).Msg(msg)
+	l.Logger.Error().Fields(getLogFields(keyVals...)).Msg(msg)
 }
 
 func (l defaultLogger) Debug(msg string, keyVals ...interface{}) {
 	l.Logger.Debug().Fields(getLogFields(keyVals...)).Msg(msg)
 }
 
 func (l defaultLogger) With(keyVals ...interface{}) Logger {
-	return defaultLogger{
+	return &defaultLogger{
```

Comment:
Since we're returning pointers now, should the methods here be operating on pointers?